frontend/src/metabase/containers/SaveQuestionModal.unit.spec.tsx

```diff
@@ -182,7 +197,7 @@ describe("SaveQuestionModal", () => {
         expect(onCreateMock).toHaveBeenCalledTimes(1);
       });
 
-      const call = onCreateMock.mock.calls[0];
+      const call: Question[] = onCreateMock.mock.calls[0];
       expect(call.length).toBe(1);
```

Comment:
Should we maybe refactor this test to use `expect(...).toHaveBeenCalledOnce()` instead?

Reply:
we have checks for the argument later in the code, so `call` is useful anyway

```
      const newQuestion = call[0];
      expect(newQuestion.id()).toBeUndefined();
      expect(newQuestion.displayName()).toBe(EXPECTED_SUGGESTED_NAME);
      expect(newQuestion.description()).toBe(null);
      expect(newQuestion.collectionId()).toBe(null);
```

I doubt I'll be able to replace it with `expect().toHaveBeenCalledWith(expect.objectContaining({}))`, so maybe it's better to keep it as it is

Reply:
We can use both ways at the same time, but fwiw I'm fine keeping it as is